.github/workflows/release.yml

```
@@ -0,0 +1,148 @@
+name: Release
+on:
+  release:
+    types: [created]
+
+jobs:
+  create-release:
+    runs-on: ubuntu-latest
+    outputs:
+      release_id: ${{ steps.create-release.outputs.result }}
```

actionlint (1.7.4)

property "create-release" is not defined in object type {get_version: {conclusion: string; outcome: string; outputs: {string => string}}}

(expression)